app-crypt/snow/snow-20130616.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
+HOMEPAGE="http://www.darkside.com.au/snow/"
+COPYRIGHT="Matthew Kwan"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="http://www.darkside.com.au/snow/$portVersionedName.tar.gz"
+CHECKSUM_SHA256="c0b71aa74ed628d121f81b1cd4ae07c2842c41cfbdf639b50291fc527c213865"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	snow = $portVersion
+	cmd:snow = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES=""
+
+BUILD_PREREQUIRES="
+	haiku_devel
```

Comment:
There has been a small change some time ago. We now prefer to have haiku_devel in BUILD_REQUIRES instead of BUILD_PREREQUIRES:
```
BUILD_REQUIRES="
	haiku_devel
	"
BUILD_PREREQUIRES="
	cmd:gcc
	cmd:make
	"
```